.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_CLIENT_ID=572926868404-jf8u20cba81glkuej1qht0sbh6ib9qpo.apps.googleusercontent.com
```

Comment:
Add this file to .gitignore,
You shouldn't commits those type of files

Reply:
Thanks!
moved secrets to Circle CI